src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -493,6 +507,78 @@ public int[] EndTime
             }
         }
 
+        /// <summary>
+        /// Get the list of the properties of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<string> PropertyList
+        {
+            get
+            {
+                if (null == _propertyList)
+                {
+                    _propertyList = new List<string>();
+                }
+
+                return _propertyList;
+            }
+        }
+
+        /// <summary>
+        /// Get the list of the destination value for each property of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<string> DestValueList
+        {
+            get
+            {
+                if (null == _destValueList)
+                {
+                    _destValueList = new List<string>();
+                }
+
+                return _destValueList;
+            }
+        }
+
+        /// <summary>
+        /// Get the list of the start time for each property of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<int> StartTimeList
+        {
+            get
+            {
+                if (null == _startTimeList)
+                {
+                    _startTimeList = new List<int>();
+                }
+
+                return _startTimeList;
+            }
+        }
+
+        /// <summary>
+        /// Get the list of end time for each property of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public List<int> EndTimeList
+        {
+            get
+            {
+                if (null == _endTimeList)
+                {
+                    _endTimeList = new List<int>();
+                }
+
+                return _endTimeList;
```

Comment:
in c#, pascal case is supposed to be used and for private member, it would be OK to use `_` prefix. However in NUI, we have decided **NOT to use prefix such as `_`, `m`, `p`, and etc.**

- NUI ground rule : https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule
- NUI coding rule : https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=191373542

could you use **PascalCase** even for private fields?